Fundamentals/Data/Content-Version-Cleanup/index.md

```diff
@@ -6,18 +6,22 @@ versionTo: 10.0.0
 
 # Content Version Cleanup
 
-When you publish a document a lot of new records are created in the database. All of the property data for each culture variant of a document are duplicated and this can mount up fast taking disk space on your database server.
+A new version is created whenever you save and publish a content item in Umbraco. This is how you can roll back to a previous version. Every saved version stores a record in the database, not only for the version but also for each content item property for that version. In a multi-lingual site, further rows are added for every culture variation. Over time this amount of data can build and swallow up the capacity of your SQL Server and slow the performance of the Umbraco backoffice.
 
 Umbraco 9.1.0 introduced a feature to clean up historic content versions (inspired by [Our.Umbraco.UnVersion](https://our.umbraco.com/packages/website-utilities/unversion/)).
 
 ## How it works
 
-The default cleanup policy is to remove all versions that are more than 4 days old except for the latest version which will be kept for 90 days.
+The default cleanup policy will:
+ - Not delete any versions created over the previous 4 days. The recent version history is preserved. See the `KeepAllVersionsNewerThanDays` setting.
+ - 'Prune' versions 4 days after they are created. The last version of a content item saved on a particular day will be kept but earlier versions from that day will be deleted.
+ - Delete all versions older than 90 days. See the `KeepLatestVersionPerDayForDays` setting.
+ - Never delete any 'published' versions.
+ - Never delete any specific versions marked as 'Prevent Cleanup' in the backoffice version history.
 
-The feature is enabled by default via configuration for new installs starting from 9.1.0 but will require to opt in for 
-those upgrading from 9.0.0.
+The feature is enabled by default via configuration for new installs starting from 9.1.0 but will require to opt in for those upgrading from 9.0.0.
```

Comment:
```suggestion
The feature is enabled by default via configuration for new installs starting from 9.1.0. It will require to opt-in for those upgrading from 9.0.0.
```